conf/local.config

```
@@ -2,34 +2,52 @@
 // BACTpipe Nextflow configuration file
 process {
     errorStrategy = 'ignore'
-    executor = 'local'  
+    executor = 'local'
     scratch = true
     stageInMode = 'copy'
```

why are we using copy staging for the local profile? I assume the local a profile will mostly be used on machines with limited resources (mostly laptops?), where you really want to limit unnecessary space waste. I think we should change this to symlink
```suggestion
    stageInMode = 'symlink'
```

---

Hi @boulund, that's a good question. I found this setting when I picked up the project, however with the latest `stub` feature being added by the https://github.com/ctmrbio/BACTpipe/pull/135#issue-554120035 , this might actually not be a big problem since people can test it with a couple genomes (no stub) and debug the file flow using the `-stub` feature. Plus keeping the settings similar to the other profiles helps in more consistent testing.

If you'd like me to update these anyhow, please let me know :)

---

I think we might as well change these to symlink, as it makes it consistent with the other profiles.

---

Cool, I've implemented this with https://github.com/ctmrbio/BACTpipe/pull/135/commits/3cc1eb3c6451732fed644b08f8ff73220685cec2

Note: `symlink` [doesn't exist](https://github.com/ctmrbio/BACTpipe/pull/135/commits/3cc1eb3c6451732fed644b08f8ff73220685cec2) for `stageOutMode` and the `stageInMode` defaults to `symlink`.

---

Great!

Yeah, that does make sense now that I think of it.